src/App.tsx

```diff
@@ -16,48 +19,21 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [state, setState] = useState<State>({
+    reverse: false,
```

Comment:
when you have boolean value its better to give name like `isReverse` , `hasReverse`

please rename